pwa-devdocs/src/tutorials/cloud-deploy/index.md

```diff
@@ -2,238 +2,198 @@
-        NODE_ENV=development yarn install --ignore-optional
-        yarn build
-        cp -p -r dist/ "$HOME/pwa"
-        cd $HOME
+Navigate or open a new terminal to _your storefront project_ and edit the `.env` file.
+Set the values for each variable to the same value as the ones you set in the previous step.
 
-        php ./vendor/bin/ece-tools build:generate
-        php ./vendor/bin/ece-tools build:transfer
+Use `yarn` or `npm` to run the project's build command.
+
+```sh
+yarn build
 ```
 
-### Add required environment variables
+This command runs the build process, which creates a `dist` folder that contains code bundles for your storefront application.
+It also copies over the static assets your application needs from your project into this folder.
 
-Use the Magento Cloud GUI or modify the `variables.env` entry in `.magento.app.yaml` to add any required environment variables.
+## Add your storefront project code
 
-The following table lists the required environment variables for the Venia storefront:
+*In your Cloud project*, create a `pwa` folder and copy into it the content inside your storefront project's `dist` folder.
 
-| Name                                 | Value                                                                                                   |
-| ------------------------------------ | ------------------------------------------------------------------------------------------------------- |
-| `CONFIG__DEFAULT__WEB__UPWARD__PATH` | `/app/pwa/upward.yml` (absolute path to UPWARD YAML configuration)                                      |
-| `NODE_ENV`                           | `production`                                                                                            |
-| `MAGENTO_BACKEND_URL`                | `https://[your-cloud-url-here]`                                                                         |
-| `CHECKOUT_BRAINTREE_TOKEN`           | `<generated token from Braintree>`                                                                      |
-| `MAGENTO_BACKEND_EDITION`            | `EE` (enable Magento Commerce features)                                                                 |
-| `IMAGE_OPTIMIZING_ORIGIN`            | `backend` (use Fastly for image optimization)                                                           |
+```sh
+mkdir pwa && cp -r <path to your storefront project>/dist/* pwa
+```
+
+If you are updating your existing storefront code, you must delete the content inside the `pwa` directory before you copy the new `dist` content to avoid keeping the old bundles the application no longer uses.
 
-## Commit modified files
+## Deploy changes
 
-Commit all changes to the following files:
+At this point in the tutorial, your Cloud project should have changes in the following files and directories:
 
 -   `.magento.app.yaml`
 -   `composer.json`
 -   `composer.lock`
+-   `pwa`
```

Comment:
I believe the `.gitignore` wouldn't pick up on this directory, would need to add these lines:

```
!/pwa
!/pwa/**
```

Reply:
Addressed in 4ba1c88